lib/pages/chat/events/sending_video_widget.dart

```diff
@@ -0,0 +1,166 @@
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:chewie/chewie.dart';
+import 'package:fluffychat/pages/chat/events/message_content_style.dart';
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+import 'package:matrix/matrix.dart';
+import 'package:video_player/video_player.dart';
+
+class SendingVideoWidget extends StatefulWidget {
+
+  final Event event;
+
+  final Uint8List bytes;
+
+  final String filePath;
+
+
+  const SendingVideoWidget({
+    super.key,
+    required this.event,
+    required this.bytes,
+    required this.filePath,
+  });
+
+  @override
+  State<SendingVideoWidget> createState() => _SendingVideoWidgetState();
+}
+
+class _SendingVideoWidgetState extends State<SendingVideoWidget> with AutomaticKeepAliveClientMixin {
+  final ValueNotifier<double> sendingFileProgressNotifier = ValueNotifier(0);
```

Comment:
`0`, `1`, `-1` what is this bro?. It's very confusing for me.

IMO: 
Why don't you specifically define its states?
For example:
enum FileProgressState {
   loading,
   success,
   failure,
}

Reply:
year, i know, I make it works first. tomorrow I will refactor